aries-site/src/examples/templates/forms/FormHeaderExample.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Select,
+  Text,
+  TextInput,
+  TextArea,
+} from 'grommet';
+
+const superPower = ['Flying', 'Sky Runner', 'invisibility'];
+const weakness = ['Fire', 'PBJ', 'Kryptonite'];
+
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={required ? `${label}*` : label}
+      required={required}
+      {...rest}
+    />
+  );
+};
+
+RequiredFormField.propTypes = {
+  required: PropTypes.bool,
+  label: PropTypes.string,
+};
+
+const FormContainer = ({ ...rest }) => {
+  return (
+    <Box background="background-front" border round="small" overflow="hidden">
+      <Box flex pad={{ horizontal: 'medium', vertical: 'medium' }} {...rest} />
+    </Box>
+  );
+};
+
+export const FormHeaderExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    name: 'Enduro',
+    superPower: 'Sky Runner',
+    nemesis: true,
+    email: 'enduro@skyrunner.io.com',
+    weakness: 'PBJ',
+  });
+
+  const onFormChange = value => {
+    setFormValues(value);
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Text size="xxlarge" weight="bold">
+            Form Header
+          </Text>
+        </Header>
+        <Box
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form value={formValues} onChange={onFormChange}>
+            <RequiredFormField
+              required
```

Comment:
When I change the name, I'm not able to get the error to resolve onSubmit or onBlur. I think we should have it resolve if people change it away from the initial error value. What do you think?

Reply:
oh yeah I didnt add this state so if someone changed it the error would go away. I had initially thought just match what figma was showing the error state.. Im conflicted with these examples since technically they would have a call to an api and then wait for a 400 or bad error so not sure what the best thing for an example is. Since the SIgn up was showing that error on submit I figured that was okay even though again with that example their state and api call would be different

Reply:
If you wanted, you could have onSubmit call `mockCreateSuperHeroAPI` and have it conditionally return an object in the form of a true response...